packages/data-model/README.md

```diff
@@ -0,0 +1,3 @@
+## Data model for Lyric
+
+This package is part of Lyric mono repo, for more details go to [main page](../../README.md)
```

Comment:
This README should still function as documentation for lyric developers, and I think should be more informative for people browsing overture packages on npmjs.

So it should contain:
- Description of what the package is (what it does in the mono repo)
- Description of what Lyric is with a link to https://github.com/overture-stack/lyric
- Highlighted message that this package probably isn't the one people want to download, with a link to the npmjs page for @overture-stack/lyric (https://www.npmjs.com/package/@overture-stack/lyric)

By "highlighted message" I mean a [block quote alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):
```md
> [!NOTE] 
> This package is likely not the one you want to use in your project, it is primarily used as a dependency within the Lyric monorepo. [@overture-stack/lyric](https://www.npmjs.com/package/@overture-stack/lyric) is what you are most likely to want to use.
```

Reply:
readme file updated